src/modules/shared-components/Navbar/index.jsx

```diff
@@ -25,24 +25,20 @@ export const Navbar = ({ isLoggedIn, onLogout }) => {
         <li>
           <Link to="/recipes/categories">Categories</Link>
         </li>
-        {isLoggedIn ? (
-          <>
-            <li>
-              <Link to="/account">Account</Link>
-            </li>
-            <li>
-              <Link to="/settings">Settings</Link>
-            </li>
 
-            <li>
-              <button onClick={onLogout}>Log out</button>
-            </li>
-          </>
-        ) : (
-          <li>
-            <Link to="/login">Login</Link>
-          </li>
-        )}
+        <li>
+          <Link to="/account">Account</Link>
+        </li>
+        <li>
+          <Link to="/settings">Settings</Link>
+        </li>
+        <li>
+          <Link to="/login">Login</Link>
+        </li>
+
+        <li>
+          <button className="buton-bt">Logout</button>
```

Comment:
1. Only show this item to logged in users. 
2. Style this menu items in the same way as other items. 
<img width="1510" alt="Screenshot 2024-10-14 at 00 08 21" src="https://github.com/user-attachments/assets/98843bbd-5ec2-48db-b145-0d43de6d835b">